### PR TITLE
Added PHP 7.1 to travis (#4889)

### PR DESCRIPTION
**Pull request by dimitri-koenig**
Should be self-explanatory

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -39,3 +42,7 @@ script:
 
 notifications:
   slack: akeneo:fDZaQeRRj1gVtXCW3f2kQAxo
+
+matrix:
+  allow_failures:
+    - php: "7.1"
```
Comment (jmleroux) [inline .travis.yml]: missing blank line

**Comment by jmleroux** [comment]
Could be interesting, thanks for this contribution :+1:

**Comment by juliensnz** [comment]
GTM after comment fix, thanks a lot :)

**Comment by grena** [comment]
GTM after comment fixed, thank you!

File: .travis.yml [hunk 2]
```diff
@@ -11,18 +12,20 @@ cache:
     - $HOME/.composer/cache/files
 
 before_install:
-    - phpenv config-rm xdebug.ini
+    - if [ "$TRAVIS_PHP_VERSION" != "7.1" ]; then
+        phpenv config-rm xdebug.ini;
```
Comment (nidup) [inline .travis.yml]: i don't understand this special case for 7.1, could you explain?

Comment (dimitri-koenig) [inline .travis.yml]: This file does not exist in the PHP 7.1 installation. Take a look at this build/test log: https://travis-ci.org/akeneo/pim-community-dev/jobs/154847084

**Comment by dimitri-koenig** [comment]
EOF line fixed

File: .travis.yml (hunk 2 above)
Comment (nidup) [inline .travis.yml]: ok thank you

**Comment by nidup** [comment]
I don't get why the following spec fails with this build change,

File: .travis.yml [hunk 3]
```diff
@@ -11,18 +12,20 @@ cache:
...
+      fi;
     - echo -e "Host github.com\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config
     - composer self-update --no-interaction
     - if [ "$TRAVIS_PHP_VERSION" = "5.6" ]; then
         echo "extension = mongo.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini;
       fi;
-    - if [ "$TRAVIS_PHP_VERSION" = "7.0" ]; then
+    - if [ "$TRAVIS_PHP_VERSION" >= "7.0" ]; then
```
Comment (nidup) [inline .travis.yml]: I think this line does not work, please by
` - if [ "$TRAVIS_PHP_VERSION" = "7.0" ] || [ "$TRAVIS_PHP_VERSION" = "7.1" ]; then`

cf https://travis-ci.org/akeneo/pim-community-dev/builds/155041419

File: .travis.yml [hunk 4]
```diff
@@ -11,18 +12,20 @@ cache:
...
         pecl install mongodb;
       fi;
 install:
     - composer update --prefer-dist --no-interaction --ignore-platform-reqs
-    - if [ "$TRAVIS_PHP_VERSION" = "7.0" ]; then
+    - if [ "$TRAVIS_PHP_VERSION" >= "7.0" ]; then
```
Comment (nidup) [inline .travis.yml]: same here, then it should be ok and I'll merge :tada:

**Comment by dimitri-koenig** [comment]
done

**Comment by nidup** [comment]
Thanks a lot @dimitri-koenig ! Don't hesitate to open another PRs will be happy to merge them! :tada:

**Pull request merged**